itou/openid_connect/models.py

```diff
@@ -112,12 +112,15 @@ def check_valid_kind(self, user, user_data_dict, is_login):
 
     def create_or_update_user(self, is_login=False):
         """
-        Create or update a user managed by another identity provider.
-         - If there is already a user with this username (user_info_dict["sub"])
-           and from the same identity provider, we update and return it.
-         - If there is already a user with the email, we return this user, provided
-           the email is not in use by another account of the same provider (some providers allow email overloading).
-         - Otherwise, we create a new user based on the data we received.
+        A user is being created or updated from information provided by an identity provider.
+        A user is globally unique with the combination of sub + SSO provider (e.g. InclusionConnect:username).
+
+        If we cannot find the user via provider + username:
+         - If the email isn't in use, we'll create a new account.
+         - We will replace a Django account with the SSO account information.
```

Comment:
We will replace a Django account **with the same email** with the SSO account information.